synfig-studio/src/gui/workarea.cpp

```diff
@@ -1749,6 +1760,7 @@ WorkArea::on_hruler_event(GdkEvent *event)
 			set_drag_mode(DRAG_GUIDE);
 			curr_guide = get_guide_list().insert(get_guide_list().begin(),{synfig::Point(((1.0/2.0)*(drawing_area->get_window()->get_width())*get_pw())+ get_window_tl()[0], 0),
 																			   synfig::Angle::rad(0)});
+			guide_highlighted = true;
```

Comment:
Why did you write this here?

Reply:
After dragging a new guide from the rule user wont be able to immediately delete the guide because guide_highlighted is set to false. By adding this line you can drag a new guide and then delete immediately without having to move the mouse. Added to the vruler for the same reason.